src/deadline/client/cli/_groups/attachment_group.py

```diff
@@ -0,0 +1,125 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+"""
+All the `deadline attachment` commands:
+    * upload
+    * download
+"""
+from __future__ import annotations
+
+import click
+import boto3
+
+from typing import Optional
+
+from .._common import _apply_cli_options_to_config, _handle_error
+from ...config import config_file
+from deadline.client.api._attachment import attachment_download as download_attachment_api
```

Comment:
How about if we added `__all__` like `deadline.api`, then we can do `from deadline.job_attachments import api`, then the usage is just `api.attachment_download`, that's the style of the library.

Reply:
Updated!